### PR TITLE
fix: Change location of the proto-generated python code

### PR DESCRIPTION
**Pull request by achals**
The subdirectory only complicates structures because the proto generated imports are not correct (sine python 3 requires absolute imports, but the proto generated imports start with things like `feast.core` instead of `feast.proto.feast.core`.

Signed-off-by: Achal Shah <achals@gmail.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: sdk/python/feast/value_type.py [hunk 1]
```diff
@@ -50,12 +61,26 @@ class ValueType(enum.Enum):
...
-ListType = Union[
-    Type[BoolList],
-    Type[BytesList],
-    Type[DoubleList],
-    Type[FloatList],
-    Type[Int32List],
-    Type[Int64List],
-    Type[StringList],
-]
+_value_type_proto_value_type_mapping = {
```
Comment (woop) [inline sdk/python/feast/value_type.py:64]: Is there a reason for these changes?

File: sdk/python/pyproject.toml [hunk 2]
```diff
@@ -2,22 +2,4 @@
 line-length = 88
 target-version = ['py37']
 include = '\.pyi?$'
-exclude = '''
```
Comment (woop) [inline sdk/python/pyproject.toml:5]: What happened to all these exclusions?

File: sdk/python/feast/diff/registry_diff.py [hunk 3]
```diff
@@ -8,16 +8,14 @@
...
 from feast.feature_service import FeatureService
 from feast.feature_view import DUMMY_ENTITY_NAME
-from feast.protos.feast.core.DataSource_pb2 import DataSource as DataSourceProto
-from feast.protos.feast.core.Entity_pb2 import Entity as EntityProto
-from feast.protos.feast.core.FeatureService_pb2 import (
-    FeatureService as FeatureServiceProto,
-)
-from feast.protos.feast.core.FeatureView_pb2 import FeatureView as FeatureViewProto
-from feast.protos.feast.core.OnDemandFeatureView_pb2 import (
+from feast.proto_core.DataSource_pb2 import DataSource as DataSourceProto
```
Comment (woop) [inline sdk/python/feast/diff/registry_diff.py:11]: I am a little bit unclear on what problem you are trying to solve with this change? Didn't we have absolute imports before and after?

**Comment by kevjumba** [comment]
@achals I think there is some merge conflicts that need to be resolved and also willem left some comments!

File: sdk/python/feast/diff/registry_diff.py (hunk 3 above)
Comment (achals) [reply sdk/python/feast/diff/registry_diff.py:11]: While we did, there was a bug that I found later and fixed in #2437.

With this PR - I just think that the subdir structure is a bit stuttery - `feast.protos.feast.*`. It's a taste thing, not a major functionality thing.

**Comment by achals** [comment]
Doesn't seem like this is causing a problem - and the value in minimal, so I'm going to close this PR for now.

**Pull request closed without merging**